.ci-support/production-testing-env.sh

```diff
@@ -15,7 +15,7 @@ set -x
 # patched by the incoming development. The following vars control the
 # production environment:
 #
-# export PRODUCTION_BRANCH=""   # The base production branch to be installed by emirge
+export PRODUCTION_BRANCH="wave-lazy-production"   # The base production branch to be installed by emirge
```

Comment:
Do we need this change for the final merge @MTCam ?

Reply:
You want main's version before merge.